app/serializers/user_post_bookmark_serializer.rb

```diff
@@ -37,6 +37,10 @@ def bookmarkable_url
     post.full_url
   end
 
+  def url_for_ui
```

Comment:
I see `bookmarkable_url` above. Can we rely on that instead?

Reply:
Yes, `url_for_ui` and `bookmarkable_url` are actually equivalent in this serializer.